report/2.Analysis/DistanceCalculation.tex

```diff
@@ -12,10 +12,7 @@ \section{Distance to target}
 
 This problem produced several different solutions.
 One solution, would be using a 3D camera, while another would be to use a \textit{ultrasonic sensor}, as presented in Section~\ref{sec:sensors}, and point it at the target.
-3D cameras are quite expensive, and not necessarily reliable, but can provide the device with data faster than the ultrasonic sensor, as this does not have to be pointed exactly at the target.
-\todo{Citation needed above}
-Using an ultrasonic sensor is the cheapest option and presumably has a higher degree of precision, but this would require the device to actually point the sensor at the target, which is quite time costly.
-\todo{Hvordan får jeg kilde på ovenstående sætning?}
+3D cameras are quite expensive and as an alternative, using an ultrasonic sensor is a cheaper option and has a higher degree of precision, but this would require the device to actually point the sensor at the target, which is time costly \cite{sensors}.
```

Comment:
```suggestion
3D cameras are quite expensive and as an alternative, using an ultrasonic sensor is a cheaper option and has a higher degree of precision, but this would require the device to actually point the sensor at the target, which is time consuming\cite{sensors}.
```